packages/components/src/responsive-toolbar-group/swipe-group.tsx

```diff
@@ -47,10 +54,15 @@ export default function SwipeGroup( {
 						key={ `button-item-${ index }` }
 						id={ `button-item-${ index }` }
 						isActive={ activeIndex === index }
+						href={ hrefList[ index ] }
 						ref={ activeIndex === index ? ref : null }
-						onClick={ () => {
+						onClick={ ( event: React.MouseEvent ) => {
 							setActiveIndex( index );
 							onClick( index );
+
+							if ( typeof hrefList[ index ] === 'string' ) {
```

Comment:
Nitpick: Might need to Yoda-ise this condition.

Reply:
@claudiucelfilip In fact, yoda condition is a rule only for PHP. The JavaScript part of WordPress Coding Standards doesn't contain that. You can find lots of `typeof xxx === ` conditions in Calypso. 😄

Reply:
Good to know. Thanks! 😁